TwoJugWaterPuzzle/00_two_jug_water_puzzle.py

```diff
@@ -0,0 +1,56 @@
+"Implementation of the two jug water puzzle."
+
+
+def gcd(a, b):
```

Comment:
Let's help the reader to understand...
```suggestion
def greatest_common_denominator(a, b):
```

Reply:
okay I will make sure this